collect_app/src/main/java/org/odk/collect/android/utilities/QuestionFontSizeProvider.java

```diff
@@ -0,0 +1,21 @@
+package org.odk.collect.android.utilities;
+
+import org.odk.collect.android.preferences.GeneralSharedPreferences;
+
+import static org.odk.collect.android.preferences.GeneralKeys.KEY_FONT_SIZE;
+
+public class QuestionFontSizeProvider {
```

Comment:
This should probably just be `QuestionFontSizeUtils` now

Reply:
Right, fixed.